src/QMCDrivers/WFOpt/QMCCostFunctionBatched.h

```diff
@@ -79,6 +79,9 @@ class QMCCostFunctionBatched : public QMCCostFunctionBase, public QMCTraits
 
   SampleStack& samples_;
 
+  // Number of samples local to each node
+  int numSamples_;
```

Comment:
Base class has NumSamples
Derived class has numSamples_.
So confusing. Put it rank_local_num_samples_